src/components/forms/common/FileUploader.tsx

```diff
@@ -3,33 +3,26 @@ import { Controller } from 'react-hook-form';
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore TODO: fix import errors
 import { MuiFileInput } from 'mui-file-input';
-import { useState } from 'react';
 
 type Props = {
 	name: string;
 	control: Control<any>;
 	label?: string;
 };
 
-//TODO: make image loading to work (value not passed via context now)
-export const FileUploader = ({ name, control, label }: Props) => {
-	const [file, setFile] = useState<null | File>(null);
-	return (
-		<Controller
-			name={name}
-			control={control}
-			render={({ field, fieldState }) => (
-				<MuiFileInput
-					{...field}
-					label={label}
-					value={file}
-					onChange={(newFile: File | null) => {
-						setFile(newFile);
-					}}
-					helperText={fieldState.invalid ? 'File is invalid' : ''}
-					error={fieldState.invalid}
-				/>
-			)}
-		/>
-	);
-};
+export const FileUploader = ({ name, control, label }: Props) => (
+	<Controller
+		name={name}
+		control={control}
+		render={({ field, fieldState }) => (
+			<MuiFileInput
+				{...field}
+				label={label}
+				accept="image/*"
```

Comment:
Toto zatím nefunguje. Nevím, jestli to ta knihova vůbec podporuje.

Reply:
Našel jsem, že `inputProps={{ accept: 'image/*' }}` funguje, ale pouze při kliknutí (v nabídce jsou pouze obrázky). Pro drag&drop to nefunguje (lze přetáhnout i pdfko). :D

Reply:
Možná bychom měli přidat nějakou validaci přetáhnutého souboru?
Můžeme zkontrolovat jednak příponu, jednak velikost, protože asi úplně nechceme velké obrázky, jinak nám pár obrázků zaplácne celou databázi.